acquia_lift.services.yml

```diff
@@ -7,6 +7,6 @@ services:
     arguments: ['@config.factory', '@path.current', '@request_stack', '@acquia_lift.service.helper.path_matcher']
   acquia_lift.service.helper.path_matcher:
     class: Drupal\acquia_lift\Service\Helper\PathMatcher
-    arguments: ['@path.alias_manager', '@path.matcher']
+    arguments: ['@path_alias.manager', '@path.matcher']
```

Comment:
it seems this is breaking our test in Drupal 8.7, so we might have to require Drupal 8.8 in our .info.

Reply:
cc @nidaismailshah